### PR TITLE
feat: add dotted lines bg for web & mobile

### PR DESCRIPTION
**Pull request by selina-kim**
## Summary by CodeRabbit

- **New Features**
	- Added new background images for different screen sizes, enhancing visual appeal.
	- Reordered components within the app for improved user experience.
	- Added a new div element between the Hero and Footer components for better layout structure.

### CODE REVIEW

File: public/dotted_line_bg_ver2.svg
Comment (err53) [inline public/dotted_line_bg_ver2.svg:1]: Why is there so much white space in this SVG file?

File: src/pages/index.astro [hunk 1]
```diff
@@ -16,16 +16,23 @@ import Footer from "../components/Footer.astro";
...
-  <Sponsors id="Sponsers" />
-  <FAQ id="FAQ" />
-  <CountdownAndLinks id="CountdownAndLinks" />
+  <div
+    class="bg-[url('/dotted_line_bg_ver2.svg')] bg-[length:auto_98%] bg-[70%_10px] bg-no-repeat md:bg-[url('/dotted_line_bg_ver1.svg')] md:bg-cover md:bg-top"
+  >
+    <WhoAreWe id="WhoAreWe" />
+    <Community id="Community" />
+    <CarouselWrapper id="Carousel" />
+    <Sponsors id="Sponsers" />
```
[the remark this answers is not in the document]
Comment (err53) [reply src/pages/index.astro]: Can we apply this change?

[the remark this answers is not in the document]
Comment (selina-kim) [reply src/pages/index.astro]: fixed

File: public/dotted_line_bg_ver2.svg
Comment (selina-kim) [reply public/dotted_line_bg_ver2.svg:1]: Got rid of the width that was defined in the svg. Is it better or does it still have too much white space..?

Comment (err53) [reply public/dotted_line_bg_ver2.svg:1]: Hmm there was a bunch of vertical space in Github's preview, seems to be gone now so should be fine I think

**Review by err53 — APPROVED** [review]

**Pull request merged**